spec/index.md

```diff
@@ -1615,6 +1617,20 @@ Example: To accept all cycles provided in a call, invoke `ic0.msg_cycles_accept(
 
     This does not trap.
 
+-   `ic0.cycles_burn : (amount_high : i64, amount_low : i64, dst : i32) -> ()`
+
+    This burns cycles from the canister. It burns as many cycles as possible, up to these constraints:
+
+    It burns no more cycles than the amount obtained by combining `amount_high` and `amount_low`. Cycles are represented by 128-bit values.
```

Comment:
What does "combining" mean here? When I saw the names of the parameters I thought these amounts were upper and lower bounds, but here it says "no more cycles than", are we adding them up?